lib/perl/Genome/Sys/Lock/FileBackend.pm

```diff
@@ -300,8 +295,7 @@ sub tempdir_for_resource {
         Carp::croak("Failed to create temp lock directory ($tempdir)");
     }
 
-    # make this readable for everyone
-    chmod(0770, $tempdir) or Carp::croak("Can't chmod 0770 path ($tempdir): $!");
+    mode($tempdir)->add_group_rwx();
```

Comment:
It probably doesn't actually matter, but do we want group execute permissions or just r/w?